mediafile.py

```diff
@@ -1754,9 +1806,20 @@ def update(self, dict):
         MP3StorageStyle('TPE2'),
         MP4StorageStyle('aART'),
         StorageStyle('ALBUM ARTIST'),
+        StorageStyle('ALBUM_ARTIST'),
         StorageStyle('ALBUMARTIST'),
         ASFStorageStyle('WM/AlbumArtist'),
     )
+    albumartists = ListMediaField(
+        MP3ListDescStorageStyle(desc=u'ALBUMARTISTS'),
+        MP3ListDescStorageStyle(desc=u'ALBUM ARTISTS'),
+        MP3ListDescStorageStyle(desc=u'ALBUM_ARTISTS'),
+        MP4ListStorageStyle('----:com.apple.iTunes:ALBUMARTISTS'),
+        ListStorageStyle('ALBUMARTISTS'),
+        ListStorageStyle('ALBUM ARTISTS'),
+        ListStorageStyle('ALBUM_ARTISTS'),
```

Comment:
I added more tags containing this data and now I'm getting:
```
E   AssertionError: Tags did not match
E     Tag albumartists: ['multival', 'test', 'multival', 'test', 'multival', 'test'] != ['multival', 'test']
```
I'm not sure what's the best way to make it work, maybe someone have some idea?

Reply:
Anyway - I'll do new PR with this change later. Now I'm waiting for main code to be merged as my other code is dependent on this PR.

Reply:
Moved to #47.